x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysTypes.java

```diff
@@ -28,15 +28,23 @@
 import static org.elasticsearch.xpack.sql.type.DataType.INTEGER;
 import static org.elasticsearch.xpack.sql.type.DataType.SHORT;
 
+/**
+ * System command designed to be used by JDBC / ODBC for column metadata.
+ * In JDBC it used for {@link DatabaseMetaData#getTypeInfo()},
+ * in ODBC for <a href="https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqlgettypeinfo-function">SQLGetTypeInfo</a>
+ */
 public class SysTypes extends Command {
 
-    public SysTypes(Location location) {
+    private final Integer type;
```

Comment:
Don't we need to adjust hashCode() and equals() to check for the field?

Reply:
Fixed.